lib/mountain_view/presenter.rb

```diff
@@ -0,0 +1,70 @@
+module MountainView
+  class Presenter
+    class_attribute :_attributes, instance_accessor: false
+    self._attributes = {}
+
+    attr_reader :slug, :properties
+    alias_method :props, :properties
```

Comment:
The component classes end up with a lot of references to the `properties` hash and I found I wanted a shorter alias. I think it does, but maybe I'm just being lazy :wink: